projects/rocprofiler-sdk/source/lib/python/rocpd/source/types.hpp

```diff
@@ -286,7 +286,7 @@ struct kernel_dispatch
     uint64_t                stack_id            = 0;
     uint64_t                parent_stack_id     = 0;
     uint64_t                corr_id             = 0;
-    uint64_t                vgpr_count          = 0;
+    uint64_t                arch_vgpr_count     = 0;
```

Comment:
```suggestion
    uint64_t                vgpr_count     = 0;
```
Sorry, it looks like they reverted back to VGPR_COUNT:
https://github.com/ROCm/rocm-systems/commit/ea891a16fdba9659aa577f8a18fdbeb7da7cab58 

You probably have to revert these :(  Sorry.

Reply:
Looks like change was made.